docs/databases/swift-kuery.html

```diff
@@ -154,91 +164,58 @@ <h2 class="heading-2"><span class="blue-text">Step 3:</span> Create a class to r
       <p>Our class needs to inherit from `Table` and have fields that match the column names of the table we created in the database.</p>
       <p>We must also provide the table name as a property.</p>
       <p>We then need to create an instance of this table to use.</p>
-      <p>Return to `KueryRoutes.swift`. Below your connection pool, add the following:</p>
-      <pre><code class="language-swift">let titles = Titles()</code></pre>
+      <p> create a connection pool of connections:</p>
+      <p>Return to `KueryRoutes.swift`. In the `App` extension, below your connection pool, add the following:</p>
+      <pre><code class="language-swift">static let bookTable = BookTable()</code></pre>
       <p>Now we're ready to start saving data to our database.</p>
       <div class="underline"></div>
       <h2 class="heading-2"><span class="blue-text">Step 4:</span> Save data to the database</h2>
-      <p>We need to modify our POST route to save the data:</p>
-      <pre><code class="language-swift">app.router.post("/kuery") { request, response, next in
-    let rows: [[Any]]
-    do {
-        let book = try request.read(as: Book.self)
-        rows = [[book.id, book.title, book.price, book.genre]]
-    } catch {
-        Log.error(error.localizedDescription)
-        return
+      <p>Inside our `insertHandler`, we need to convert our book to an Array of Arrays of type `Any`:</p>
```

Comment:
I think you need some context here about what we're doing and why. Maybe something like "Now we're going to save some book data, which is posted to the '/kuery' route, into our database." if that is what we're doing....